C/plugins/storage/postgres/CMakeLists.txt

```diff
@@ -6,14 +6,24 @@ set(CMAKE_CXX_FLAGS_DEBUG "-O0 -ggdb")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 set(STORAGE_COMMON_LIB storage-common-lib)
 
+# Handle Postgres on RedHat/CentOS
+set(FOGLAMP_RH_POSTGRES "/opt/rh/rh-postgresql96/root/usr/include" CACHE INTERNAL "")
```

Comment:
Can we avoid giving absolute path here and in other files in this PR?

Reply:
ok, working to improve..

Reply:
improvement done, testing.